lib/src/pages/leaderboard/leader_board_page.dart

```diff
@@ -16,9 +17,17 @@ class _LeaderBoardPage extends State<LeaderBoardPage> {
   }
 
   String _gameName = '';
+  List scoreList = [];
   _LeaderBoardPage() {
     GameService.instance.getGames().then(
           (value) => setState(() {
+            ScoreService.instance.findAllByGameUid(value.data.first.uid).then(
```

Comment:
der Code sollte nicht in der setState stehen, die Werte kommen ja asynchron an. Wenn du die scoreList in der UI benutzt, musst du das setzen innerhalb eines setState machen.

Reply:
WidgetsBinding.instance?.addPostFrameCallback